src/nvim/window.c

```diff
@@ -1178,22 +1178,24 @@ int win_split_ins(int size, int flags, win_T *new_wp, int dir)
   int need_status = 0;
   int new_size = size;
   bool new_in_layout = (new_wp == NULL || new_wp->w_floating);
+  bool vertical = flags & WSP_VERT;
+  bool toplevel = flags & (WSP_TOP | WSP_BOT);
```

Comment:
What is the point of these two booleans? They make code less consistent as the value of `flags` still needs to be check for other flags.